src/approx_const.rs

```diff
@@ -13,14 +12,25 @@ declare_lint! {
      is found; suggests to use the constant"
 }
 
-const KNOWN_CONSTS : &'static [(f64, &'static str)] = &[(f64::E, "E"), (f64::FRAC_1_PI, "FRAC_1_PI"),
-    (f64::FRAC_1_SQRT_2, "FRAC_1_SQRT_2"), (f64::FRAC_2_PI, "FRAC_2_PI"),
-    (f64::FRAC_2_SQRT_PI, "FRAC_2_SQRT_PI"), (f64::FRAC_PI_2, "FRAC_PI_2"), (f64::FRAC_PI_3, "FRAC_PI_3"),
-    (f64::FRAC_PI_4, "FRAC_PI_4"), (f64::FRAC_PI_6, "FRAC_PI_6"), (f64::FRAC_PI_8, "FRAC_PI_8"),
-    (f64::LN_10, "LN_10"), (f64::LN_2, "LN_2"), (f64::LOG10_E, "LOG10_E"), (f64::LOG2_E, "LOG2_E"),
-    (f64::PI, "PI"), (f64::SQRT_2, "SQRT_2")];
-
-const EPSILON_DIVISOR : f64 = 8192f64; //TODO: test to find a good value
+// Tuples are of the form (name, lower_bound, upper_bound)
```

Comment:
I initially defined the intervals as `[a, b)` (as is typical), but `utils/dogfood.sh` [didn't like the inclusive lower bound](https://travis-ci.org/Manishearth/rust-clippy/builds/86937120#L228). Any ideas on how to work around this?